.github/workflows/benchmark.yml

```diff
@@ -9,6 +9,9 @@ on:
       - "papers/**"
       - "rfcs/**"
       - "*.md"
+  pull_request:
+    branches:
+      - benchmark
```

Comment:
This can be removed.

Reply:
Yeah I'll remove this after the workflow runs